src/go/cmd/api_view.go

```diff
@@ -13,12 +13,32 @@ import (
 
 // CreateAPIView generates the output file that the API view tool uses.
 func CreateAPIView(pkgDir, outputDir string) error {
-	m, err := NewModule(pkgDir)
+	review, err := createReview(pkgDir)
 	if err != nil {
 		panic(err)
 	}
+	if outputDir == "." {
+		outputDir = ""
+	} else if !strings.HasSuffix(outputDir, "/") {
+		outputDir = fmt.Sprintf("%s/", outputDir)
+	}
+	filename := fmt.Sprintf("%s%s.json", outputDir, review.Name)
```

Comment:
Another option, which might avoid the need to normalize path separators, is to use [`filepath.Join`](https://pkg.go.dev/path/filepath#Join).